app/index.js

```diff
@@ -247,7 +248,13 @@ module.exports = generators.Base.extend({
       }
     }
   },
-  install: function () {
-    if(!this.options['skip-install']) this.installDependencies();
+  install: function() {
+    if (!this.options['skip-install']) {
+      return this.installDependencies(function() {
+        return spawn('npm', ['run', 'test:coverage'], {
```

Comment:
This is looking really good thanks! The test time isn't too bad either.

I'm going to take a proper look through the PR as soon as I can, hopefully tomorrow night and merge it